.eslintignore

```diff
@@ -1,3 +1,5 @@
 coverage
 dist
 node_modules
+fixtures
+packages/rule-webpack-config/README.md
```

Comment:
Why is ESLint looking at this file?

Reply:
Eslint look at all the md files and check the js/ts code.
In this case, I have an intentional error in the code, and I don't want Eslint to fail :)

Reply:
ok.